content/administration/install/cdn.rst

```diff
@@ -1,80 +1,109 @@
+Status` to :guilabel:`active` to engage the zone. For the :guilabel:`Zone Type` set the value to
+:guilabel:`Pull`, and then, finally, under the :guilabel:`Pull Settings`, enter the
+:guilabel:`Origin URL`— this address should be the full Odoo database :abbr:`URL (Uniform Resource
+Locator)`.
 
-When creating the zone, enable the CORS option in the
-:guilabel:`advanced features` submenu. (more on that later)
+.. example::
+   Use ``https://yourdatabase.odoo.com`` and replace the *yourdatabase* subdomain prefix with the
+   actual name of the database. A custom :abbr:`URL (Uniform Resource Locator)` can be used, as
+   well, in place of the Odoo subdomain that was provided to the database.
 
-.. image:: cdn/keycdn_enable_CORS.png
-   :class: img-fluid
+.. image:: cdn/keycdn-zone.png
+   :align: center
+   :alt: KeyCDN's Zone configuration page.
 
-Once done, you'll have to wait a bit while KeyCDN_ is crawling your website.
+Under the :guilabel:`General Settings` heading below the zone form, click the :guilabel:`Show all
+settings` button to expand the zone options. This should be the last option on the page. After
+expanding the :guilabel:`General Settings` ensure that the :guilabel:`CORS` option is
+:guilabel:`enabled`.
 
-.. image:: cdn/keycdn_progressbar.png
-   :class: img-fluid
+Next, scroll to the bottom of the zone configuration page and :guilabel:`Save` the changes. KeyCDN
+will indicate that the new zone will be deployed. This can take about 10 minutes.
 
-.. note:: a new URL has been generated for your Zone, in this case it is
-          ``http://pulltest-b49.kxcdn.com``
+.. image:: cdn/zone-url.png
+   :align: center
+   :alt: KeyCDN deploying the new Zone.
 
-Step 2: Configure the odoo instance with your zone
---------------------------------------------------
+.. note::
+   A new :guilabel:`Zone URL` has been generated for your Zone, in this example it is
+   ``pulltest-xxxxx.kxcdn.com``. This value will differ for each database.
 
-In the Odoo back end, go to the :guilabel:`Website Settings`: menu, then
-activate the CDN support and copy/paste your zone URL in the
-:guilabel:`CDN Base URL` field. This field is only visible and configurable if
-you have developer mode activated.
+Copy this :guilabel:`Zone URL` to a text editor for later, as it will be used in the next steps.
 
-.. image:: cdn/odoo_cdn_base_url.png
-   :class: img-fluid
+Configure the Odoo instance with the new zone
+---------------------------------------------
 
-Now your website is using the CDN for the resources matching the
-:guilabel:`CDN filters` regular expressions.
+In the Odoo :guilabel:`Website` app, go to the :menuselection:`Settings` and then activate the
+:guilabel:`Content Delivery Network (CDN)` setting and copy/paste the :guilabel:`Zone URL` value
+from the earlier step into the :guilabel:`CDN Base URL` field. This field is only visible and
+configurable when :doc:`../../applications/general/developer_mode` is activated.
```

Comment:
When referencing another page with the `doc` directive, it's usually safer to force a label if you are using the page title in your sentence. Otherwise, your sentence might look weird when someone renames that page: "configurable when Find out how to enable dev mode is activated."